app/src/forms/email/emailService.js

```diff
@@ -112,6 +113,67 @@ const buildEmailTemplate = async (formId, formSubmissionId, emailType, referer,
   };
 };
 
+/** Helper function used to build the email template based on email type and contents for reminder */
+const buildEmailTemplateFormForReminder = async (form, emailType, users, report, referer) => {
+  let configData = {};
+  const closeDate = moment(report.dates.closeDate).format('MMM. D, YYYY');
+  const subject = 'CHEFS Submission Reminder';
+  const message = (report.dates.closeDate) ? `This email is to inform you that the ${form.name} form is now open for submissions and will stay open until ${closeDate}. Please ensure to complete your submission before the period is closed.
+  Thank you.` : `This email is to inform you that the ${form.name} form is now open for submissions.
```

Comment:
This should say `This email is to inform you that the ${form.name} form is now open for submission.`